client/header/activity-panel/panels/inbox/index.js

```diff
@@ -30,6 +30,24 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	getScreenName() {
+		let screenName = '';
+		const URLparams = Object.fromEntries(
```

Comment:
This doesn't follow normal JS naming conventions, should be `urlParams`

Reply:
Sounds good, I changed the variable name in the commit [867b5b7](https://github.com/woocommerce/woocommerce-admin/pull/4320/commits/867b5b7dafc56d4630d757fcdb2f2a8792788ff3)